include/mata/nfa.hh

```diff
@@ -574,7 +574,28 @@ public:
      */
     void get_one_letter_aut(Nfa& result) const;
 
-    void print_to_DOT(std::ostream &outputStream) const;
+    /**
+     * @brief Prints the automaton in DOT format
+     * 
+     * @return automaton in DOT format 
+     */
+    std::string print_to_DOT() const;
+    /**
+     * @brief Prints the automaton to the output stream in DOT format
+     */
+    void print_to_DOT(std::ostream &output) const;
+    /**
+     * @brief Prints the automaton in mata format
+     * If you need to parse the automaton again, use IntAlphabet in construct()
```

Comment:
The `@brief` needs an empty line after the brief description. As per [Doxygen documentation](https://www.doxygen.nl/manual/commands.html#cmdbrief).
```suggestion
     * @brief Prints the automaton in mata format
     *
     * If you need to parse the automaton again, use IntAlphabet in construct()
```